src/api/uploads/MultiputUpload.js

```diff
@@ -143,22 +138,41 @@ class MultiputUpload extends BaseMultiput {
         this.overwrite = overwrite;
         this.fileId = fileId;
 
-        this.createSession();
+        this.makePreflightRequest();
     }
 
+    /**
+     * Update uploadHost with preflight response and return the base uploadUrl
+     *
+     * @private
+     * @param {Object} response
+     * @param {Object} [response.data]
+     * @return {string}
+     */
+    getBaseUploadUrlFromPreflightResponse = ({ data }: { data: { upload_url?: string } }) => {
+        if (!data || !data.upload_url) {
+            return this.getBaseUploadUrl();
+        }
+
+        const splitUrl = data.upload_url.split('/');
+        this.uploadHost = `${splitUrl[0]}//${splitUrl[2]}`;
```

Comment:
two // ?
Can you put an example as comments

Reply:
`splitUrl[0]` is the protocol part (e.g., https:), `splitUrl[2]` is hostname (e.g., www.box.com)

Sure I'll add comments